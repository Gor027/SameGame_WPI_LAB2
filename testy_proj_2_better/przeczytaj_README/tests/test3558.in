  VV6 V V6 666V V6V66 6V6VV6 6
V6 6 VV  VV VV666VV6V6V VVVV6V  V6V6 6 666 VV6 V66  V666  6 66V6V V  6 66  V6666
 VV6 VV  VV   6 V 66 VV  6V66 V6V 6V6   6VV 6
VVV  6     V   666 66     V6VVV 6 6V 6
V V  6V 66 6VV V6V 6 VV VV   66V V6VV66 V V66VVV6VV6 6 VV
6
V6 6 V    V6V6V   6 VV VVV 6  6V  VVV V6  V66  V 666V V666   66 
V  6666 6 V  66V6V6  66 6 V6   V  V6V    VV6V    6VVVV6V  V  66V6 V V6V VV6 6
V6V VV    66 66 V6   6 666 V6 VV  6666 6 
6V VVVV666V6666V V66 6 6666V66V6 VV6 
V66 6  VV    V V  VV6V  V666V6 V  66 666 66V   6V6V V6V6V
6VV 6VVVV6 
6 V66V
 VV 66V6VVV6 6 V66 V66666V 6  6V6VV   6VVVV 66VV V 6V6V
  V6  V V  6V VV6 VV 6  V 66VVV6
 V6V V V
  66V666  VVV   V
   6V   VV 66V 6V6VVV6 VVVVV 6 V V 6V 6 V 666 6
 V VVV 6
