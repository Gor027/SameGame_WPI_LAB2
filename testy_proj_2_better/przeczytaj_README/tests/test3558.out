     V
     V
  VV 6                     6
 6V666          V    6  6  V
 VV66VV    6 6  6  6 V  VV 6   6  V6
VVV6V6V V  V V 6V6V6 V  VVV6   V  6V
V6V6VVV 66V66VV66VV6 V  V6V6 6VV  6V
V66V6VV66VV6V66V666666  6VV66VVV  6V      V V
6VV66V6V6VV6V66V6V66V66V6VVV6V6V  VV6     VV6    6
V6V66VVVV6V6V6VVVV6666VV66VVV666 6V6V66666V66  V V  V666
VVVV66V6V666666V666VVVV6V66666VVVV66V6V66VV66  V 6V6V6VV
VVV6VVVVV6V6VV6V6VVV66V6666666VVVVV6V666VV66V6VV6666VV66V   6V
6V66V6V6VVVVV66VV6VV6666VV66VV666V666666V6VV6VV666VVV6VV6 6 666VV 6  6   6
VV6VVV66VVV66VV6V6VVV66VVVVVV6VV6VV6V66VVV66666VVVV6V6V6V6V666V6VV6V6VV6V6666
